plugins/aws-appmesh/plugin/plugin.go

```diff
@@ -24,7 +24,7 @@ const (
 	pluginName = "aws-appmesh"
 
 	// logFilePath is the path to the plugin's log file.
-	logFilePath = "/log/aws-appmesh.log"
```

Comment:
I think this will break ECS agent. Can you please confirm if the plugin continues to execute properly with ECS agent, with this change?

Reply:
It looks like the ECS agent is overriding this path via the VPC_CNI_LOG_FILE env variable anyway, so this change shouldn't break it. But I am going to remove this file from the PR just in case. It isn't relevant to this PR. I'll follow up with the ECS agent team later to see if there are other bugs here.